src/api/manager/inventory.ts

```diff
@@ -20,3 +21,18 @@ export async function getAudioMapping(id: ObjectId): Promise<IResponse> {
       throw `Could not find audio mapping for source: ${id.toString()}`;
     })) as IResponse;
 }
+
+export async function deleteInventorySourceItem(id: string): Promise<void> {
+  const db = await getDatabase();
+
+  const document = await db
```

Comment:
I believe you only used this for development right? So should be enough to just call `.deleteOne`?